Jenkins/alignment/cram_no_bam.yaml

```diff
@@ -0,0 +1,6 @@
+# tab-separated file of the samples
+samples: samples.tsv
+
+
+# how to sort bam
+cram_no_bram: true
```

Comment:
typo: should be `cram_no_bam`

also, the comment could be clearer, maybe `# store aligned reads in cram format`?

Reply:
Good catch.